### PR TITLE
CI for Android fixed

### PR DESCRIPTION
**Pull request by dryganets**
Fix for android github CI.

## Motivation
Github CI doesn't work without this change.

## Test Plan
CI passes.

## Release Notes
[INTERNAL] Android CI fix.

### CODE REVIEW

**Comment by dryganets** [comment]
I rolled back to the commit where iOS tv worked to make all unit tests pass.
tvOS CI is broken in the master.

**Comment by hramos** [comment]
Thanks for fixing this. It was reverted internally due to a misunderstanding because usually Yoga wouldn't need these DEFS. Let's land this for now as it fixes OSS CI until a better long term fix can be implemented.

File: ReactCommon/yoga/BUCK [hunk 1]
```diff
@@ -1,4 +1,6 @@
-fb_xplat_cxx_library(
+include_defs("//ReactCommon/DEFS")
+
+rn_xplat_cxx_library(
```
Comment (hramos) [inline ReactCommon/yoga/BUCK]: Why `rn_` and not `fb_`?

File: ReactCommon/DEFS [hunk 2]
```diff
@@ -2,8 +2,7 @@
 
 # Building is not supported in OSS right now
 def rn_xplat_cxx_library(name, **kwargs):
-  cxx_library(name = name)
-
+  cxx_library(name, **kwargs)
```
Comment (hramos) [inline ReactCommon/DEFS]: What is this change for?

Comment (dryganets) [reply ReactCommon/DEFS]: I'm still playing around. The android tests are failing randomly.
This fixed the fetch for me on the master.

Comment (dryganets) [reply ReactCommon/DEFS]: As I don't know all the steps needed to run tests I'm just pushing the branch and see the results.
Then fixing the related part of the script locally.

File: ReactCommon/yoga/BUCK (hunk 1 above)
Comment (dryganets) [reply ReactCommon/yoga/BUCK]: fb is not defined in common.

**Comment by dryganets** [comment]
There is one more android fail:

File does not exist: buck-out/gen/ReactAndroid/src/main/third-party/java/robolectric3/robolectric/android-all-4.1.2_r1-robolectric-0.jar
java.lang.IllegalArgumentException: File does not exist: buck-out/gen/ReactAndroid/src/main/third-party/java/robolectric3/robolectric/android-all-4.1.2_r1-robolectric-0.jar

**Comment by dryganets** [comment]
All robolectric dependencies by some reason are spread across set of binary-jar directories.
This causes the failure.

I guess it is a result of the BUCK tool update.

**Comment by dryganets** [comment]
@hramos this time it should build for Android.
Somebody needs to fix iosTv linking for x86_64.

**Comment by dryganets** [comment]
yey!
Android passed.
I think it is ready for merge.

**Comment by dryganets** [comment]
This is a commit which broke  tvTests:
c491b2223313676bd4900de7a8c70a10051fa9f0

**Comment by dryganets** [comment]
This PR is related and could be closed now:
https://github.com/facebook/react-native/pull/17389

**Pull request closed without merging**